tools/qt-builder/README.md

```diff
@@ -1,10 +1,6 @@
 # General
-This document describes the process to build Qt 5.12.3.
-Note that there are several patches.  
-* The first (to qfloat16.h) is needed to compile QT 5.12.3 on Visual Studio 2017 due to a bug in Visual Studio (*bitset* will not compile.  Note that there is a change in CMakeLists.txt to support this.  
-* The second patch is to OpenSL ES audio and allow audio echo cancelllation on Android.
-* The third is a patch to QScriptEngine to prevent crashes in QScriptEnginePrivate::reportAdditionalMemoryCost, during garbage collection.  See https://bugreports.qt.io/browse/QTBUG-76176
-* The fourth is a patch which fixes video playback on WebEngineViews on mac.  See https://bugreports.qt.io/browse/QTBUG-70967
+This document describes the process to build Qt 5.51.2.
+
```

Comment:
It may be too early to remove all the patches. The QScriptEngine one is merged, but I'm not 100% sure about the rest.

Reply:
Oh yeah. I was told that we don't need patched Qt anymore, but I guess "we" meant us Linux users.

Reply:
Yes, that was Linux only, in the context of "can we build against system Qt?".

@ctrlaltdavid has been doing work on upgrading Qt on windows, perhaps he can chime in with what patches are necessary on Windows if any. The patch referenced VS 2017, is that still relevant?

@ArcadeFever From looking at https://bugreports.qt.io/browse/QTBUG-70967 it looks like that fix should also be in 5.15.2, so we can use an unpatched Qt on Mac as well?

That leaves an Android patch that may need looking into, and might still be necessary since there was no QTBUG associated with it.